lib/lightning/attempts/attempt.ex

```diff
@@ -17,6 +17,23 @@ defmodule Lightning.Attempt do
   alias Lightning.Workflows.Trigger
   # alias Lightning.Workflows.Node
 
+  @final_states [
+    :success,
```

Comment:
this is the only one that isn't in the past tense. why? would it make more sense to call it either:
1. `succeeded` or
2. `completed` ?

Reply:
> this is the only one that isn't in the past tense. why? would it make more sense to call it either:
> 
> 1. `succeeded` or
> 2. `completed` ?

Just kept the use of `success`. Would you mind if we change it later when we stabilize the codebase?

Reply:
lol, good idea!